pytest/tests/sanity/congestion_control.py

```diff
@@ -194,6 +205,19 @@ def __check_txs(self, node: BaseNode):
         self.assertGreater(accepted_count, 0)
         self.assertGreater(rejected_count, 0)
 
+    def __check_tx(self, result):
+        self.assertIn('result', result)
+
+        status = result['result']['final_execution_status']
+        self.assertIn(status, GOOD_FINAL_EXECUTION_STATUS)
+
+        # The common reason for this failure is `MethodResolveError` which
+        # happens if the test contract is built without the `test_features`
+        # feature. Occasionally the contract is not rebuilt properly and cargo
+        # clean is needed.
+        status = result['result']['status']
+        self.assertIn('SuccessValue', status)
```

Comment:
It would be nice to add the explanation about test_features as the third argument to `assertIn` so that it gets printed out along with the assertion error.